src/index.js

```diff
@@ -58,11 +58,13 @@ export default class Client {
     return url;
   }
 
-  _urlencode = function(obj) {
+  _urlencode(obj) {
     let str = [];
     for (let key in obj) {
       str.push(encodeURIComponent(p) + "=" + encodeURIComponent(obj[p]));
```

Comment:
where this `p` variable is defined?

Reply:
oh, thanks for this, it should be key, when copying my test code from the browser console